### PR TITLE
Unsupport nested-JAR style JAR plugins.

### PR DESCRIPTION
**Pull request by kamatama41**
For #1110 
All codes that relate to`embeddedJarPathsInNestedJar` in `PluginClassLoader` are removed.

### CODE REVIEW

File: embulk-core/src/main/java/org/embulk/plugin/PluginClassLoader.java [hunk 1]
```diff
@@ -518,400 +327,8 @@ public InputStream getInputStream() throws IOException {
...
-            }
-        }
-        if (lastException != null) {
-            throw new ClassNotFoundException(className, lastException);
-        } else {
-            throw new ClassNotFoundException(className);
-        }
-    }
-
     private InputStream getResourceAsStreamFromChild(final String resourceName) {
```
Comment (dmikurube) [inline embulk-core/src/main/java/org/embulk/plugin/PluginClassLoader.java]: We may be able to remove this method to change the caller side to call `super.getResourceAsStream(resourceName)` directly.

Also, the caller side `getResourceAsStream` may be removed. `URLClassLoader#getResourceAsStream` is actually calling `getResource` to find the resource as below.
http://hg.openjdk.java.net/jdk8/jdk8/jdk/file/687fd7c7986d/src/share/classes/java/net/URLClassLoader.java#l232

`getResource` is overridden in `PluginClassLoader` in the same way where the overridden `getResourceAsStream` is doing.

File: embulk-core/src/main/java/org/embulk/plugin/PluginClassLoader.java [hunk 2]
```diff
@@ -301,7 +194,7 @@ public void addUrl(URL url) {
      */
     @Override
     public URL findResource(final String resourceName) {
```
[the remark this answers is not in the document]
Comment (kamatama41) [reply embulk-core/src/main/java/org/embulk/plugin/PluginClassLoader.java]: Yes, I overlooked both `if` and `else` is the same logic 😅, fixed in e55755c

File: embulk-core/src/main/java/org/embulk/plugin/PluginClassLoader.java [hunk 3]
```diff
@@ -338,32 +218,18 @@ public URL run() {
      */
     @Override
     public Enumeration<URL> findResources(final String resourceName) throws IOException {
```
[the remark this answers is not in the document]
Comment (kamatama41) [reply embulk-core/src/main/java/org/embulk/plugin/PluginClassLoader.java]: Same as above, fixed in e55755c

File: embulk-core/src/main/java/org/embulk/plugin/PluginClassLoader.java (hunk 1 above)
Comment (kamatama41) [reply embulk-core/src/main/java/org/embulk/plugin/PluginClassLoader.java]: I just removed the method and call `super.getResourceAsStream(resourceName)` directly in ca688e9

Comment (dmikurube) [reply embulk-core/src/main/java/org/embulk/plugin/PluginClassLoader.java]: Thanks!

Plus, as a straight inference looking in `URLClassLoader.java`, I guessed we can remove the overridden `PluginClassLoader#getResourceAsStream` as well.

What the overridden `PluginClassLoader#getResourceAsStream` does is almost the same as the overridden `PluginClassLoader#getResource`, and then just loading it as `InputStream` in addition.

The same operation could be done just by the non-overridden `URLClassLoader#getResourceAsStream` calling `getResource`, where `getResource` is actually overridden as `PluginClassLoader#getResource`.

Comment (kamatama41) [reply embulk-core/src/main/java/org/embulk/plugin/PluginClassLoader.java]: I see. Removed it in 90ab1f6

**Comment by kamatama41** [comment]
Thanks. I found another unused variable and removed it in df33bdd.

**Pull request merged**